src/Domain/Entity/Unfetched/UnfetchedNetwork.php

```diff
@@ -12,7 +13,8 @@ public function __construct()
         parent::__construct(
             new NullNid(),
             '',
-            new UnfetchedImage()
+            new UnfetchedImage(),
+            new Options()
```

Comment:
Should be UnfetchedOptions, as we don't know what the options are, rather than "we know them any they're empty"